src/main/java/ca/corefacility/bioinformatics/irida/processing/concatenate/impl/SingleEndSequenceFileConcatenator.java

```diff
@@ -58,7 +60,11 @@ public SingleEndSequenceFile concatenateFiles(List<? extends SequencingObject> t
 
 			SequenceFile forwardSequenceFile = single.getSequenceFile();
 
-			iridaFileStorageUtility.appendToFile(tempFile, forwardSequenceFile);
+			try {
+				IridaFiles.appendToFile(tempFile, forwardSequenceFile);
```

Comment:
Yeah I'm kinda iffy on this.  I feel like anything that *modifies* a file should maybe use the `iridaFileStorageUtility` dependency.  I'm not sure it really matters, but it feels a bit outside of `IridaFiles` responsibility.  Thoughts?

Reply:
Yeah definitely. Makes more sense not to have it in there. Removed in 97e0014e1c5aa5584ad4b4f07bf67f3d54fbaf70